easybuild/easyconfigs/i/igraph/igraph-0.10.3-foss-2022a.eb

```diff
@@ -0,0 +1,51 @@
+# Author: Denis Krišťák (INUITS)
+# Modified: Jasper Grimm (UoY)
+
+easyblock = 'CMakeMake'
+
+name = 'igraph'
+version = '0.10.3'
+
+homepage = 'https://igraph.org'
+description = """igraph is a collection of network analysis tools with the emphasis on
+efficiency, portability and ease of use. igraph is open source and free. igraph can be
+programmed in R, Python and C/C++."""
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://github.com/igraph/igraph/releases/download/%(version)s']
+sources = ['%(name)s-%(version)s.tar.gz']
+checksums = ['5f72398c7847bb167f85159b7a9fe1fe69ce0f241c5de5d30b2b347f9dc3f7c6']
+
+builddependencies = [
+    ('Autotools', '20220317'),
+    ('pkgconf', '1.8.0'),
+    ('CMake', '3.24.3'),
+]
+
+dependencies = [
+    ('GLPK', '5.0'),
+    ('libxml2', '2.9.13'),
+    ('zlib', '1.2.12'),
+    ('SuiteSparse', '5.13.0', '-METIS-5.1.0'),
+    ('arpack-ng', '3.8.0'),
+]
+
+# Build static and shared libraries
+configopts = ["-DBUILD_SHARED_LIBS=OFF", "-DBUILD_SHARED_LIBS=ON"]
+
+# extra folder `igraph` is wrong, since all other software expects it to be in ./include folder directly
+postinstallcmds = [
+    'mv %(installdir)s/include/igraph/* %(installdir)s/include/ && rmdir %(installdir)s/include/igraph ',
+]
+
+_includes = ['include/igraph%s.h' % x for x in ['', '_blas', '_constants', '_lapack', '_types', '_version']]
```

Comment:
Not sure why these specific headers are singled out. Normally one will include only `igraph.h`—the rest are there only to keep things manageable during development, and no guarantees are given about how they may change between versions.

Reply:
I guess we can trim this down to just `igraph.h` then